### DISCUSSION TITLE
UnicodeDecodeError: 'utf-8' codec can't decode bytes in position 25-26: unexpected end of data

### DISCUSSION THREAD

**Discussion by YYH211** [dashingsoft/pyarmor#1908] [General]
when i use this code ：pyarmor -d gen -O D:\desktop\dist --enable-jit -r ./
error with below
UnicodeDecodeError: 'utf-8' codec can't decode bytes in position 25-26: unexpected end of data

**Comment by jondy** [comment]
Please search "encoding" in the online doc to find solutions.

**Reply by YYH211** [reply]
Can you tell me the maximum file size that this trial version can confuse？ thanks

**Comment by jondy** [comment]
No exact value.
